The surveyor measures the following lengths:

<table>
    <tr><td></td><td>x<sub>i</sub></td><td>y<sub>i</sub></td></tr>
    <tr><td>i=1</td><td>2.698</td><td>1.252</td></tr>
    <tr><td>i=2</td><td>4.217</td><td>1.894</td></tr>
    <tr><td>i=3 (at origin)</td><td>2.115</td><td>0.913</td></tr>
    <tr><td>i=4</td><td>1.997</td><td>0.907</td></tr>
</table>

<image src="stal.png">

The measured points are not quite on a line. There is clearly a linear correlation.

In [2]:
x=r([2.698,4.217,2.115,1.997])

In [3]:
y=r([1.252,1.894,0.913,0.907])

In [4]:
r=r.cor(x,y)

In [5]:
r

[1] 0.9976906

In [9]:
L=matrix([[2.698,1.252],[4.217,1.894],[2.115,0.913],[1.997,0.907]])

In [10]:
L

[ 2.69800000000000  1.25200000000000]
[ 4.21700000000000  1.89400000000000]
[ 2.11500000000000 0.913000000000000]
[ 1.99700000000000 0.907000000000000]

In [11]:
var('a','b')

(a, b)

In [12]:
model(x)=a*x+b

In [16]:
find_fit(L,model)

[a == 0.4535097299807165, b == -0.008712942690074499]

What I just did in a program called Sage Math, you can do using the formulas for $r$, the slope and the $y$-intercept of the linear regression line. The results are $r\approx{}0.9977$ (which will pass any hypothesis test for linear correlation), and $y=0.4535x-0.008713$.

At the origin, we measured $x_{3}=2.115$ and $y_{3}=0.913$. Using the regression line equation, one of these measurement becomes redundant. $$\begin{array}{rcl}x&=&2.115-\epsilon_{1} \\ 0.4535x&=&0.913-\epsilon_{2}\end{array}$$

The YAVE setup is $$\left[\begin{array}{c}2.115 \\ 0.913\end{array}\right]-\left[\begin{array}{c}1 \\ 0.4535\end{array}\right][\hat{x}_{3}]=\left[\begin{array}{c}\epsilon_{1} \\ \epsilon_{2}\end{array}\right]$$

$\hat{x}_{3}$ is the least-squares adjusted value for the measurement $x_{3}$.

In [21]:
A=matrix([[1],[0.4535097299807165]])

In [22]:
A

[ 1.00000000000000]
[0.453509729980717]

In [23]:
Y=matrix([[2.115],[0.913]])

In [24]:
Y

[ 2.11500000000000]
[0.913000000000000]

In [28]:
x3hat=(A.transpose()*A).inverse()*A.transpose()*Y

In [29]:
x3hat

[2.09763212829814]

In [30]:
y3hat=0.4535097299807165*x3hat-0.008712942690074499

In [31]:
y3hat

[0.942583637413290]

In [32]:
y3hat/x3hat

[0.449356026110275]

These are the least-squares adjusted measurements from the origin. Consequently, the $y$-intercept of the green line is $0.9426$ and the slope is $0.4494$.